pytest/tests/mocknet/sharded_bm.py

```diff
@@ -0,0 +1,338 @@
+"""
+This script is used to run a sharded benchmark on a forknet.
+"""
+
+from argparse import ArgumentParser
+import os
+import sys
+import json
+import copy
+import pathlib
+import subprocess
+
+import time
+from types import SimpleNamespace
+from mirror import CommandContext, get_nodes_status, init_cmd, new_test_cmd, \
+    reset_cmd, run_env_cmd, run_remote_cmd, run_remote_upload_file, \
+    start_nodes_cmd, stop_nodes_cmd, update_binaries_cmd
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+from configured_logger import logger
+
+# cspell:words BENCHNET
+
+# TODO: consider moving local directory to pytest.
+LOCAL_BENCHNET_DIR = "../benchmarks/sharded-bm"
+
+REMOTE_BENCHNET_DIR = "/home/ubuntu/bench"
+NEAR_HOME = "/home/ubuntu/.near"
+CONFIG_PATH = f"{NEAR_HOME}/config.json"
```

Comment:
mini nit: Maybe make it consistent and add or remove REMOTE everywhere.

Reply:
I'll use `SOURCE_BENCHNET_DIR` instead of "local" and assume everything else is "remote".